requirements/base.txt

```diff
@@ -2,49 +2,62 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --no-index --output-file requirements/base.txt requirements/base.in
+#    pip-compile --output-file base.txt base.in
```

Comment:
not sure what happened here, but if you run `bin/compile_dependencies.cmd`, this should leave the header untouched (the `requirements/` bit is stripped off for some reason?)

Reply:
I just used generate, but will take a peek at what happened with all the downgrades